### PR TITLE
mozillavpn: add page

### PR DESCRIPTION
**Pull request by navarroaxel**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):** 2.5.0

### CODE REVIEW

**Review by CleanMachine1 — APPROVED** [review]

File: pages/common/mozillavpn.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+> A Virtual Private Network from the makers of Firefox.
+> More information: <https://github.com/mozilla-mobile/mozilla-vpn-client/wiki/Command-line-interface>.
+
+- Log in with interactive prompt:
+
+`mozillavpn login`
+
+- Connect to Mozilla VPN:
+
+`mozillavpn activate`
```
Comment (mfrw) [inline pages/common/mozillavpn.md:12]: Wondering if the VPN get activated for the current shell or the whole internet session (whatever goes out of the NIC) ?
Should we document that as well ?

Comment (navarroaxel) [reply pages/common/mozillavpn.md:12]: For the whole system. I guess it's not easy to setup a proxy just for the current bash session

File: pages/common/mozillavpn.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
+# mozillavpn
+
+> A Virtual Private Network from the makers of Firefox.
+> More information: <https://github.com/mozilla-mobile/mozilla-vpn-client/wiki/Command-line-interface>.
+
+- Log in with interactive prompt:
```
Comment (marchersimon) [inline pages/common/mozillavpn.md:6]:
```suggestion
- Log in with an interactive prompt:
```

File: pages/common/mozillavpn.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+`mozillavpn login`
+
+- Connect to Mozilla VPN:
+
+`mozillavpn activate`
+
+- Display connection status:
```
Comment (marchersimon) [inline pages/common/mozillavpn.md:14]:
```suggestion
- Display the connection status:
```

**Review by marchersimon — APPROVED** [review]
Thanks @navarroaxel

**Review by mfrw — APPROVED** [review]
LGTM :)
Thank you @navarroaxel

**Pull request merged**